apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestLogger.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Google.Cloud.Spanner.V1.Logging;
+using Xunit.Abstractions;
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests
+{
+    internal class TestLogger : DefaultLogger
+    {
+        private static Lazy<TestLogger> s_Instance = new Lazy<TestLogger>(
```

Comment:
FYI, you can still add a breakpoint with my proposed line by placing the cursor anywhere in the object creation expression and pressing F9 (or whatever your breakpoint shortcut in VS is).